src/app/server.settings/server.settings.component.ts

```diff
@@ -189,11 +198,38 @@ export class ServerSettingsComponent implements OnInit, AfterViewInit{
             this.licenseStatus = "Invalid";
             console.log("invalid license");
 
+            if (this.currentLicence.status == this.licenseServerRequestError){
+
+                licenseErrorTitle = "Could Not Connect To License Server"
+                licenseStatusExplaination = "Please Check Your Connection"
+            }
+
+            var statusJson:string = this.currentLicence.status;
+
+            JSON.parse(statusJson, (key, value) => {
+
+                if (key == "result" && value == this.allLicensesUsedError ){
+
+                    licenseStatusExplaination = "Your license is granted to another instance";
+                }
+
+                if (key == "refreshInterval" ){
+
+                    licenseStatusExplaination += ", please close your other instance, wait "+ value + " minutes and try again.";
```

Comment:
If the order has changed somehow, it will show
Please close your instance.... Your license is 
Granted to another instance

Reply:
removed first explanation assignment because "refreshInterval" is fetched in the same JSON packet together with "allLicensesUsedError". There is no need to control "allLicensesUsedError", getting "refreshInterval" ensures that "allLicensesUsedError"